### ISSUE TITLE
Code: 500 Publishing Error (Portal for ArcGIS)

### ISSUE THREAD

**Issue by nitinmv** [Esri/Survey123Community#241]
Getting a Code: 500 error when trying to publish a XLSX form with Cascading Select. 

The message box has "Error:  Creating feature class failed."

Using version 1.1.19 version and the form is being published to Portal.

**Comment by survey123** [comment]
Could you please send your XLSX file to survey123@esri.com

**Comment by nitinmv** [comment]
[SampleCascadeSelectSurvey.xlsx](https://github.com/Esri/Survey123Community/files/35044/SampleCascadeSelectSurvey.xlsx)

Have attached a sample.

**Comment by survey123** [comment]
I was able to successfully publish your XLSX  to an ArcGIS Online organization. We will look into this more closely as this seems to be an issue specific to Portal for ArcGIS. I will ask Esri Tech Support to follow up on this.  Thanks.

**Comment by nitinmv** [comment]
Thanks. This looks to be specific to Portal. Did test and works when publishing to ArcGIS Online organisational account.

**Comment by survey123** [comment]
Looking into this more closely. I found a couple things:
 -Questions of type 'end' don't publish to Portal for ArcGIS because 'end' is a reserved word in the PostgreSQL-based ArcGIS Data Store.  We will need to figure how to address this. In the meantime, if you remove the 'end' question it will help.
 -There is also an issue related to your Cascading Select. Not sure exactly what is going on, but if you disable the option to transform select_ones into domains, it will also help.

Removing the 'end' question and disabling the creation of domains in select_ones at publishing time I was able to successfully publish the survey into Portal for ArcGIS.

**Comment by nitinmv** [comment]
Cheers. It works.

**Comment by bowlesgsrcx** [comment]
I am getting the code 500 error - Inconsistency in field definitions on a survey that worked in AGOL but not now in Portal. Where do i disable the option to transform select_ones into domains?

Thanks

**Comment by survey123** [comment]
Hi. Can you please include your form so we can have a look? Would like to know why the Form is not publishing so we can help.
We disabled the option to not create domains (we could get it back if required).

**Comment by bowlesgsrcx** [comment]
sure thing. here is the form.

[ES Compliance.xlsx](https://github.com/Esri/Survey123Community/files/87720/ES.Compliance.xlsx)

**Comment by survey123** [comment]
Definitively, it is the Cascading select that breaks publishing on Portal. 
You will need to remove it for now.
We will discuss next week how to address this.

**Comment by bowlesgsrcx** [comment]
Is there another field type that will provide the same functionality as the cascading selects? that is the main reason we selected survey123.

Thanks,
--gary

**Comment by survey123** [comment]
Not really. @bowlesgsrcx  I will send a custom build of Connect to Jacob in around 60 minutes which does not create domains. Lets see if that works for you while we address the problem for good.

**Comment by bowlesgsrcx** [comment]
Sounds good. Thanks for your help.

**Comment by bowlesgsrcx** [comment]
Here is a form with cascading selects that does publish to portal.
[Meter Calibration.xlsx](https://github.com/Esri/Survey123Community/files/87889/Meter.Calibration.xlsx)

**Comment by survey123** [comment]
@bowlesgsrcx  setup sent to Jacob Bowles

**Comment by nitinmv** [comment]
Hi, Getting a 'Code: 500 Read timed out error' when publishing using the latest release of  Survey 123 Connect  (Version 1.6.11)

![image](https://cloud.githubusercontent.com/assets/15861898/17685392/87c57724-63b7-11e6-94f9-ebf9be269ef0.png)

It gave the same error when publishing the sample forms that comes with the App.  

For using the latest Connect with Portal 10.3.1 

- Do we need to use a 'custom build of Connect' as suggested in the above thread?
- Also the latest release does not have the option of 'disabling the creation of domains' at publishing time.  Given that most of the forms were successfully published without domains using the previous release of Connect, does the latest release support this option?

Thanks,
